README.md

```diff
@@ -216,15 +216,14 @@ import optunahub
 
 
 def objective(trial: optuna.Trial) -> float:
-    x = trial.suggest_float("x", 0, 1)
+  x = trial.suggest_float("x", -5, 5)
+  y = trial.suggest_float("y", -5, 5)
+  return x**2 + y**2
```

Comment:
nit; according to optuna's coding style wiki mentioning https://peps.python.org/pep-0008/, one indent is four spaces:

```suggestion
    x = trial.suggest_float("x", -5, 5)
    y = trial.suggest_float("y", -5, 5)
    return x**2 + y**2
```